src/components/Header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import Container from "@mui/material/Container";
```

Comment:
these imports can be combined like `import { AppBar, ToolBar, Container } from "@mui/material"`